### ISSUE TITLE
5.0 OpenCL dialog for macbook pro 2017

### ISSUE THREAD

**Issue by pkienzle** [SasView/sasview#1367]
Sasview 5.0, Macbook Pro 15" 2017, 10.14.6 Majave

* SasView GPU selector is not recognizing that Radeon Pro 560 supports double precision.  Maybe this is a sasmodels error?
* OpenCL on Intel CPU fails on some models (this is a sasmodels issue), and that causes the GPU selection dialog to hang (which is a sasview issue).

**Comment by pkienzle** [comment]
Error on message:
```
14:43:10 - INFO: building bcc_paracrystal-float64-946DBCA9 for OpenCL Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz
Unhandled error in Deferred:

Traceback (most recent call last):
  File "/Users/pkienzle/conda/envs/bumps/lib/python3.7/site-packages/qt5reactor.py", line 271, in _iterate
    self.runUntilCurrent()
  File "/Users/pkienzle/conda/envs/bumps/lib/python3.7/site-packages/twisted/internet/base.py", line 875, in runUntilCurrent
    f(*a, **kw)
  File "/Users/pkienzle/conda/envs/bumps/lib/python3.7/site-packages/twisted/internet/defer.py", line 501, in errback
    self._startRunCallbacks(fail)
  File "/Users/pkienzle/conda/envs/bumps/lib/python3.7/site-packages/twisted/internet/defer.py", line 568, in _startRunCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/Users/pkienzle/conda/envs/bumps/lib/python3.7/site-packages/twisted/internet/defer.py", line 654, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/Users/pkienzle/Source/sasview/src/sas/qtgui/Perspectives/Fitting/GPUOptions.py", line 240, in testFail
    self.testingFailedSignal.emit(msg)
builtins.TypeError: GPUOptions.testingFailedSignal[str].emit(): argument 1 has unexpected type 'Failure'
```

**Comment by rozyczko** [comment]
Fixed the SasView error shown above.

**Comment by butlerpd** [comment]
@pkienzle Has the SasView portion of this been fixed then? If so can we close this?
Will temporarily assign to 5.0.3.

**Comment by butlerpd** [comment]
I believer this would have been fixed by 5.0.3 so moving it and closing now.
